gulpfile.js

```diff
@@ -65,9 +64,6 @@ async function bundleWidgetJS() {
 async function compileWidgetSass() {
   const pluginsProcess = [
     discardComments(),
-    customPrefix({
```

Comment:
This is still being done but it's being done in SASS so that it's easier to apply it to some styles but not to other styles.